### PR TITLE
[Automation] Initial release

### PR DESCRIPTION
**Pull request by arrownj**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Automation

File: src/automation/azext_automation/manual/_help.py [hunk 1]
```diff
@@ -0,0 +1,221 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+helps['automation account'] = """
```
Comment (yungezz) [inline src/automation/azext_automation/manual/_help.py:10]: just curious, why this isn't generated but manual?

**Review by yungezz — APPROVED** [review]
LGTM

File: src/automation/azext_automation/manual/_help.py (hunk 1 above)
Comment (arrownj) [reply src/automation/azext_automation/manual/_help.py:10]: I hide `account` and `job` in the generated code. The reason is that we need to update `account create` command (remove some parameters). Currently we can't hide `create_or_update` operations via codegen's .md file, thus the default `account create` will always have some parameters which should be hidden. So currently the only solution is to hide all the command group and manually implement it.

**Comment by arrownj** [comment]
The readme.md file changes are in this [PR](https://github.com/Azure/azure-rest-api-specs/pull/11648/files).

**Pull request merged**